File: AppTesteAndroid/app/src/main/java/com/accenture/project/apptesteandroid/remoteData/LoginService.java
package com.accenture.project.apptesteandroid.remoteData;

import com.accenture.project.apptesteandroid.model.LoginRequest;
import com.accenture.project.apptesteandroid.model.LoginResponse;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LoginService {

    @POST("login")
    Call<LoginResponse> login(@Body LoginRequest user);
}

File: BankApp/app/src/main/java/com/example/bankapp/features/login/data/service/LoginService.kt
package com.example.bankapp.features.login.data.service

import com.example.bankapp.features.login.model.LoginRequest
import com.example.bankapp.features.login.model.LoginResponse
import io.reactivex.Single
import retrofit2.http.Body
import retrofit2.http.POST

interface LoginService {
    @POST("login")
    fun requestLogin(
        @Body
        loginRequest: LoginRequest
    ): Single<LoginResponse>
}

